crates/re_renderer/shader/rectangle_fs.wgsl

```diff
@@ -40,53 +40,77 @@ fn decode_color(sampled_value: Vec4) -> Vec4 {
     return rgba;
 }
 
+/// Takes a floating point texel coordinate and outputs a neighrest neighbor
+fn to_clamp_to_border_sample(coord: Vec2, texture_dimension: Vec2) -> IVec2 {
```

Comment:
```suggestion
fn clamped_nearest_neighbor(coord: Vec2, texture_dimension: Vec2) -> IVec2 {
```

🤷 

"clamp-to-border" sometimes means "use a special border color for samples outside the texture", as opposed to "clamp-to-edge" which is what is happening here

![image](https://github.com/rerun-io/rerun/assets/1148717/a1011bbe-e20d-4e90-9c4e-98f907a3e36f)

Reply:
you're right, I used the wrong term!